src/components/editor/editor-pane/upload-handler.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Editor } from 'codemirror'
+import i18n from 'i18next'
+import { uploadFile } from '../../../api/media'
+import { store } from '../../../redux'
+
+export const handleUpload = (file: File, editor: Editor): void => {
+  if (!file) {
+    return
+  }
+  const mimeType = file.type
```

Comment:
You already fetch the mime-type here. Instead of allowing arbitrary uploads, please check that your list of supported mime-types includes this one.

Reply:
which one?

Reply:
```typescript
import { supportedMimeTypes } from 'tool-bar/utils/upload-image-mimetypes.ts'
if (!supportedMimeTypes.includes(mimeType)) {
  return
}
```

Reply:
done